### DISCUSSION TITLE
Projects Auto-archive status filter

### DISCUSSION THREAD

**Discussion by doronguttman** [community/community#49722] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

The auto-archive feature on projects is great!
But it would be better if it could only archive items of a certain status (e.g. `status:Done`) - currently the status filter is not supported.

Thanks!

**Comment by rileybroughten** [comment]
Thank you @doronguttman for the feedback! Noted this for the team 👍 There is also a separate [discussion](https://github.com/orgs/community/discussions/36466) for feedback for the auto-archive workflow, so I will note this there and close this on out!
